lua/core/paramset.lua

```diff
@@ -107,6 +107,11 @@ function ParamSet:add_separator(name)
   self.count = self.count + 1
   self.group = self.group - 1
   self.hidden[self.count] = false
+  if name == nil then
+    self.lookup["separator_"..self.count] = self.count
```

Comment:
i do think autonaming the separator is really not very helpful while scripting. if someone wants to hide a sep, we should require that the sep be named

otherwise, cool!